src/bp/core/bpfs/drivers/disk-driver.ts

```diff
@@ -163,6 +201,13 @@ export class DiskStorageDriver implements StorageDriver {
       .value()
   }
 
+  private _hash(str: string): string {
+    return crypto
+      .createHash('sha256')
+      .update(str)
+      .digest('hex')
+  }
```

Comment:
This piece of code is duplicated in a couple of place.. please search for `calculateHash`